composer.libraries.json

```diff
@@ -17,6 +18,21 @@
                   "installer-name": "farmOS-map"
                 }
             }
+        },
+        "farmos-timeline": {
+            "type": "package",
+            "package": {
+                "name": "farmos/farmos-timeline",
+                "version": "0.0.1",
```

Comment:
I'd like to make a new release of the timeline library - we got a new release of the gantt library that should make a few things easier!